rollbar/resource_notification_test.go

```diff
@@ -165,6 +165,41 @@ func (s *AccSuite) TestNotificationUpdate() {
 	})
 }
 
+func (s *AccSuite) TestNotificationCreateSpecialEmail() {
+	notificationResourceName := "rollbar_notification.email_notification"
+	// language=hcl
+	config := `
+		resource "rollbar_notification" "email_notification" {
```

Comment:
Minor formatting question-- would it be OK to remove the leading whitespace on this line to have the `resource` block a bit more legible?

Reply:
done, this is really a string, so we would need to do our formatting by hand. gofmt does not format multiple line strings really well.